testsuite/integration-arquillian/servers/app-server/jboss/wildfly/src/main/java/org/keycloak/testsuite/arquillian/wildfly/container/WildflyAppServerProvider.java

```diff
@@ -85,6 +85,7 @@ private Node standaloneContainer() {
         createChild("jbossHome", appServerHome);
         createChild("javaHome", appServerJavaHome);
         createChild("jbossArguments", 
+                "-Djboss.server.base.dir=" + appServerHome + "/standalone-test " +
```

Comment:
I think this is causing `org.keycloak.testsuite.adapter.servlet.DemoServletsAdapterTest#historyOfAccessResourceTest` to fail somehow. There is an assertion that checks if the server logs contains a specific string, but the server logs obtained in the test is pointing to `standalone` profile and not `standalone-test`.

Reply:
@pedroigor thank you for finding the cause. I wasn't aware of this. I've updated the test to find correct log file.